utils/transformData.js

```diff
@@ -58,6 +58,8 @@ export const parseKeywords = (keywordsString) => {
 
     return trimmedArray.filter((item) => item.length > 0);
   }
+
+  return '';
```

Comment:
Do not change returned structure/type. Return either null and handle it, or return empty array